weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundContext.tsx

```diff
@@ -0,0 +1,38 @@
+import {createContext, useContext} from 'react';
+
+export type PlaygroundContextType = {
+  isPlayground: boolean;
+  deleteMessage: (messageIndex: number, responseIndexes?: number[]) => void;
+  editMessage: (messageIndex: number, newMessage: any) => void;
+  deleteChoice: (choiceIndex: number) => void;
+  editChoice: (choiceIndex: number, newChoice: any) => void;
+  addMessage: (newMessage: any) => void;
```

Comment:
I feel like add -> edit -> delete is a more natural order to list these. Can these `any` be made more restrictive?